### PR TITLE
get_news working successfully now.

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#345 
Closes: # (issue number)
#345 
## Proposed changes

Brief description of what is fixed or changed
First I resolved the issue of local variable 'news_' referenced before assignment but after that, it gave another error due to the wrong assignment of API Key I got my API key entered but didn't. I have removed unwanted imports and decreased complexity. I have resolved all issues regarding -wgn command now working successfully.
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Running Successfully
Screenshot from 2020-05-19 20-07-02

WhatsApp Image 2020-05-19 at 8 08 28 PM

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ wplay/get_news.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950&bid=18200727)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950&bid=18200727)

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***about_changer*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
-***get_news*** is a script to get all types of news in your whatsapp group.
+***get_news*** is a script to get all types of news in your whatsapp group.Visit https://newsapi.org/ to get your own API key and paste it in newsapi = NewsApiClient(api_key="YOUR API KEY")
```
Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

File: wplay/get_news.py [hunk 2]
```diff
@@ -18,10 +16,10 @@
...
 # endregion
 
-
-load_dotenv()
-newsapi = NewsApiClient(api_key=os.getenv("NEWS_API_KEY"))
-
+'''
+Visit https://newsapi.org/ to get your own API key.
+'''
+newsapi = NewsApiClient(api_key="YOUR API KEY")
```
Comment (rpotter12) [inline wplay/get_news.py:22]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950&bid=18200727)

File: README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***about_changer*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
-***get_news*** is a script to get all types of news in your whatsapp group.
+***get_news*** is a script to get all types of news in your whatsapp group.Visit [https://newsapi.org/] to get your own API key and paste it in newsapi = NewsApiClient(api_key="YOUR API KEY")
```
Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950&bid=18200727)

File: README.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***about_changer*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
-***get_news*** is a script to get all types of news in your whatsapp group.
+***get_news*** is a script to get all types of news in your whatsapp group.Visit [foo]: https://newsapi.org/ to get your own API key and paste it in newsapi = NewsApiClient(api_key="YOUR API KEY")
```
Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950&bid=18200727)

File: README.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***about_changer*** is a script to change the about section of your profile. User can select personal quote, news, quotes, etc.
 
-***get_news*** is a script to get all types of news in your whatsapp group.
+***get_news*** is a script to get all types of news in your whatsapp group.Visit 'https://newsapi.org/' to get your own API key and paste it in newsapi = NewsApiClient(api_key="YOUR API KEY")
```
Comment (rpotter12) [inline README.md:37]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5550950)

**Comment by SinghHimadri** [comment]
@rpotter12 Please review my PR.

**Pull request merged**